app/ts/schema.ts

```diff
@@ -50,8 +51,12 @@ export function createUnitParser(decimals?: bigint): funtypes.ParsedValue<funtyp
 
 export const AddressParser: funtypes.ParsedValue<funtypes.String, string>['config'] = {
 	parse: value => {
-		if (!isAddress(value)) return { success: false, message: `${value} is not a valid address string.` }
-		else return { success: true, value: getAddress(value) }
+		try {
+			// convert hex string to bigint to strip possible leading zeros
+			const maybeAddress = BigInt(value).toString(16)
```

Comment:
This isn't correct as it won't correctly pad out all 40 characters if the address starts with a 0.  I believe we have a helper function for this somewhere, which is probably better to use than inlining all of it.
```suggestion
			const maybeAddress = `0x${BigInt(value).toString(16).padStart(40, '0')}`
```

Reply:
I added a condition to the suggestion above with `value && BigInt(value).toString(16).padStart(40, '0')` since value can also be an empty string that will false validate as a valid address if padded with zeros